fluff/__init__.py

```diff
@@ -15,6 +15,9 @@
 import fluff.util
 
 
+default_app_config = 'fluff.app_config.FluffAppConfig'
```

Comment:
yea, I didn't really want to go this route but also didn't want to have to coordinate with an update to settings.py